src/diamond/handler/riemann.py

```diff
@@ -88,9 +91,13 @@ def process(self, metric):
         try:
             self.client.send_event(event)
         except Exception, e:
-            self.log.error(
+            self.log.exception(
                 "RiemannHandler: Error sending event to Riemann: %s", e)
 
+            # Re-open the transport. The riemann server may have gone away
+            self._close()
+            self._connect()
```

Comment:
I want to deploy this and see the type of exceptions that come up. We can handle the conditions in the future.